src/Iconv/README.md

```diff
@@ -0,0 +1,17 @@
+Symfony Polyfill / Iconv
+========================
+
+The [`symfony/polyfill`](https://github.com/symfony/polyfill) library backports
+features found in the latest PHP versions and provides compatibility layers for
+some extensions and functions.
+
+This `iconv` component provides a native PHP implementation of the
+[php.net/iconv](http://php.net/iconv) functions
+(short of [`ob_iconv_handler`](http://php.net/manual/en/function.ob-iconv-handler.php)).
+
+More information can be found in the [root README](../../README.md).
```

Comment:
please don't use such relative link: it will be broken when seeing the readme in the subtree repos (which is where there readme are mostly useful IMO, for people finding these repos)